compute_tools/src/compute.rs

```diff
@@ -268,21 +268,25 @@ impl ComputeNode {
         // In this case we need to connect with old `zenith_admin`name
         // and create new user. We cannot simply rename connected user,
         // but we can create a new one and grant it all privileges.
-        let mut client = match Client::connect(&self.connstr, NoTls) {
+        let mut client = match Client::connect(self.connstr.as_str(), NoTls) {
             Err(e) => {
                 info!(
                     "cannot connect to postgres: {}, retrying with `zenith_admin` username",
                     e
                 );
-                let zenith_admin_connstr = self.connstr.replacen("cloud_admin", "zenith_admin", 1);
+                let mut new_connstr = self.connstr.clone();
```

Comment:
This is actually not new connstr, but old one :) I'd really keep the long old name `zenith_admin_connstr` as it makes it easier to grep and delete later